core/tests/protractor_desktop/embedding.js

```diff
@@ -257,6 +257,10 @@ describe('Embedding', function() {
       browser.switchTo().frame(driver.findElement(
         by.css('.protractor-test-embedded-exploration > iframe')));
       waitFor.pageToFullyLoad();
+
+      // TODO(kevinlee12): Remove this sleep.
+      browser.sleep(6000);
```

Comment:
This is a temporary measure to see if the timeout is the issue.

Reply:
and will cause the linter test to fail.

Reply:
Linter will no longer fail on this since I've added this file to the exception list.

Reply:
TODOs should always link to an issue number. Please take the time to explain really clearly / verbosely in the issue what still needs to be done.